src/dsp_tools/commands/ingest_xmlupload/user_information.py

```diff
@@ -0,0 +1,121 @@
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Any
+
+import pandas as pd
+
+# pylint: disable=too-few-public-methods
+
+
+separator = "\n    "
+list_separator = "\n    - "
```

Comment:
Now I see that these 2 lines are repeated in 3 files. Then it would be better to define them only once, and import in the other files.

Reply:
Would that not be a little overkill? I like it to be consistent, but I am hesitant to import something where it is not crucial that it is identical.

Reply:
From a general point of view, I think that repeating stuff is less scalable. Once we have many files, and each repeats this, it can be inadvertently modified in 1 place. So I think defining it only once is the cleaner way. But I agree that the separators aren't that important, so I leave it up to you :-)

Reply:
just to throw it out there: maybe the fact that these separators are repeated in multiple files is actually an indication that something else is repeated too, namely the logic of creating bulletpoint list strings; and that may be something to put in a little util function?

Reply:
That is something we could certainly look at but I think it it outside of the scope of this PR...

Reply:
fair enough